extension/src/toolbar/components/InteractionMode.tsx

```diff
@@ -0,0 +1,26 @@
+import { useAtomValue, useSetAtom } from "jotai";
+import { useEffect } from "react";
+
+import { Tool, toolAtom } from "../../prototypes/view/states/tool";
+import { InteractionModeType } from "../../shared/reearth/interactionMode";
+import { reearthInteractionModeAtom } from "../../shared/states/interactionMode";
+
+const TOOL_TO_INTERACTIONMODE: Record<Tool, InteractionModeType> = {
+  hand: "move",
+  select: "selection",
+  sketch: "sketch",
+  story: "default", // TODO: Check later
+  pedestrian: "move", // TODO: Check later
+};
+
+export const InteractionMode = () => {
+  const tool = useAtomValue(toolAtom);
+  const setInteractionMode = useSetAtom(reearthInteractionModeAtom);
```

Comment:
Is there any reason you don't set this up and export from your states directory (to keep ALL Jotai use in one place) and instead do it directly here?

Reply:
I think it's difficult to bundle Jotai's logic to one place, because Jotai's atoms are used in different places in different use case. So I think it will makes the code be complicating if we make functions for these atoms.🤔

Reply:
But if we have it in one directory doesn't that simplify things? In this case, if we have any issues around InteractionMode saving/not saving/not updating we can look in the state folder. 
Especially using useSetAtom here, which specifically updates without rerendering, I feel like it'd be hard to troubleshoot for someone other than you who put this InteractionMode here.
Let me know if that makes sense or you still feel like it should be like this and we can maybe discuss more tomorrow.

Reply:
Do you think it's what you expected?
[refactor: use hooks](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/14/commits/5b63e27319c9d9c90a4068b8db9a96ddf6c8717f)